modules/nwtc-library/tests/test_NWTC_IO_FileInfo.F90

```diff
@@ -73,12 +74,46 @@ subroutine test_initfileinfo2()
         "line 3",  &
         "line 4"   &
     /)
+    ! make the last character a + so a trim does not reduce this string length
+    tmpstring=input_strings(5)
```

Comment:
Can you explain this more? Why do we need to add the `a` at the end?

Reply:
The parsing in the `InitFileInfo_FromStringArray` was modified to check if the line after stripping of trailing comments was too long instead of checking the length of the array passed in.  This allows us to keep super long comments on a line without issue.  So, to test that this worked, I had to add the `a` at the end of the string so that the trimmed length was long enough.

Reply:
Ah got it, thanks.